review-tools/ghmerge

```diff
@@ -100,12 +111,12 @@ else
 fi
 
 if [ "$BUILD" == "yes" ] ; then
-    echo Rebuilding
-    ( opensslbuild 2>&1 ) | tail -3
+    echo Rebuilding...
+    ( CC="$CC" opensslbuild 2>&1 ) | tail -3
 fi
 
 while true ; do
-    echo -n "Enter YES to push or NO to abort: "
+    echo -n "Enter YES/yes to push or NO/no to abort: "
```

Comment:
I'm not sure this change is all that useful.

BTW: `pick-to-branch` also has the same yes/no question.

Reply:
I just wanted to make clear that the simpler-to-type lower-case variants can also be used.
How if I instead `s/YES/yes/` and `s/NO/no/` in this tool and in `review-tools/pick-to-branch`?

Reply:
I've just implemented the latter.

Reply:
That's a better suggestion.  I end up typing just _y_ usually.
The intent of the question was to stop accidental continuation, but I do answer _no_ from time to time.